challenge_03_loops_conditionals/loops-conditionals/while/exercise_00.js

```diff
@@ -0,0 +1,19 @@
+/*
+    1. Create a variable called "loggedIn" that starts off as false.
+    2. Create a while loop that loops while "loggedIn" is false.
+    3. Each loop, print out: "Incorrect login credentials"
+    3. After 3 loop iterations, change "loggedIn" to be true.
+    4. After the loops is done print out: "Sucessfully logged in!"
+
+    HINT1: Be careful for an infinite loop!
+*/
+let loggedIn = false;
+let i = 0;
```

Comment:
You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable